solace-monitoring-mcp-server/solace_monitoring_mcp_server.py

```diff
@@ -375,6 +424,16 @@ def _build_input_schema(self, parameters: List[Dict[str, Any]], request_body: Op
         properties = {}
         required = []
 
+        # Add broker_alias parameter if multiple brokers are configured
+        if len(self.config.broker_aliases) > 1:
```

Comment:
Problem: If SOLACE_BROKERS_ALIAS is set to a single alias (e.g. "broker_a"), the code treats this as the "multi-broker" branch (it populates self.brokers but does not set self.default_broker_alias unless SOLACE_BROKER_DEFAULT is set). _build_input_schema only adds a broker_alias parameter when len(broker_aliases) > 1, so tools will not require broker_alias. At runtime _invoke_tool will use self.config.default_broker_alias (None) and raise ValueError("Broker alias not specified and no default broker is configured.").